recipes/default.rb

```diff
@@ -24,7 +24,7 @@
 
 # If there are no components that need artifact deployment,
 # don't invoke apply_amps
-apply_amps = true
+apply_amps = false
```

Comment:
I'd rather do `apply_amps = node['alfresco']['apply_amps']` , then define `default['alfresco']['apply_amps'] = true` in `attributes/default.rb`; this way you can simply override the default value in your Chef JSON to disable AMP installation